doc/radosgw/dynamicresharding.rst

```diff
@@ -143,3 +143,30 @@ The command to do so is:
 
 As a convenience wrapper, if the ``--bucket`` argument is dropped then this
 command will try and fix lifecycle policies for all the buckets in the cluster.
+
+Object Expirer fixes
+--------------------
+
+Similar to the Lifecycle fixes, swift object expiry on resharded buckets on
+older clusters that had an expiry time before the cluster was upgraded with
+fixes will have dropped entries off the log pool and no longer deleted the
+object. While object expiry for dates in the future will be handled correctly by
+an upgraded RGW, for objects where delete_at times are in the past and still
+present in the cluster can be found out and deleted with the following commands.
```

Comment:
Alternative wording for consideration:

> Objects subject to Swift object expiration on older clusters may have
> been dropped from the log pool and never deleted after the bucket was
> resharded. This would happen if their expiration time was before the
> cluster was upgraded, but if their expiration was after the upgrade
> the objects would be correctly handled. To manage these expire-stale
> objects, radosgw-admin provides two new subcommands.

Reply:
Better yet:

> Objects subject to Swift object expiration on older clusters may have
> been dropped from the log pool and never deleted after the bucket was
> resharded. This would happen if their expiration time was before the
> cluster was upgraded, but if their expiration was after the upgrade
> the objects would be correctly handled. To manage these expire-stale
> objects, radosgw-admin provides two subcommands.

Reply:
:+1: